### PR TITLE
Elasticsearch unit test

### PR DESCRIPTION
**Pull request by amolnin**

### CODE REVIEW

File: common-util/pom.xml [hunk 1]
```diff
@@ -108,6 +108,10 @@
 			<artifactId>resteasy-jackson2-provider</artifactId>
 			<version>3.1.3.Final</version>
 			<exclusions>
+				<exclusion>
```
Comment (bvinayakumar) [inline common-util/pom.xml]: Format code changes.

File: common-util/src/main/java/org/sunbird/common/models/util/ActorOperations.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
  */
 public enum ActorOperations {
   ENROLL_COURSE("enrollCourse"),
+  UNENROLL_COURSE("unenrollCourse"),
```
Comment (bvinayakumar) [inline common-util/src/main/java/org/sunbird/common/models/util/ActorOperations.java]: UNENROLL_COURSE is there in release-1.11 branch. Are you not having latest release 1.11 changes in your branch?

File: elasticsearch-util/pom.xml [hunk 3]
```diff
@@ -19,12 +19,7 @@
 			<groupId>org.elasticsearch.client</groupId>
 			<artifactId>transport</artifactId>
 			<version>5.4.3</version>
-                        <exclusions>
-                                <exclusion>
-                                  <groupId>joda-time</groupId>
-                                  <artifactId>joda-time</artifactId>
-                                </exclusion>
-                        </exclusions>
+
```
Comment (bvinayakumar) [inline elasticsearch-util/pom.xml]: Why this exclusion is removed?

**Comment by amolnin** [comment]
Will remerge the changes using latest code in the branch - there were some conflicts becuase of improper merge.

File: elasticsearch-util/pom.xml (hunk 3 above)
Comment (amolnin) [reply elasticsearch-util/pom.xml]: This is a run-time dependency of elasticsearch TransportClient. If this is put off, we are not able to run the test-cases - which fails with NoClassDefFoundError. 

java.lang.NoClassDefFoundError: org/joda/time/ReadablePeriod
	at org.elasticsearch.client.transport.TransportClient.<clinit>(TransportClient.java:87)
	at sun.reflect.GeneratedSerializationConstructorAccessor9.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.objenesis.instantiator.sun.SunReflectionFactoryInstantiator.newInstance(SunReflectionFactoryInstantiator.java:45)
	at org.objenesis.ObjenesisBase.newInstance(ObjenesisBase.java:73)
	at org.mockito.internal.creation.instance.ObjenesisInstantiator.newInstance(ObjenesisInstantiator.java:14)
	at org.powermock.api.mockito.repackaged.ClassImposterizer.createProxy(ClassImposterizer.java:144)
	at org.powermock.api.mockito.repackaged.ClassImposterizer.imposterise(ClassImposterizer.java:59)
	at org.powermock.api.mockito.internal.mockcreation.MockCreator.createMethodInvocationControl(MockCreator.java:122)
	at org.powermock.api.mockito.internal.mockcreation.MockCreator.createMock(MockCreator.java:70)
	at org.powermock.api.mockito.internal.mockcreation.MockCreator.mock(MockCreator.java:47)
	at org.powermock.api.mockito.PowerMockito.mock(PowerMockito.java:144)
	at org.sunbird.common.ElasticSearchUtilTest.initBeforeTest(ElasticSearchUtilTest.java:112)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.internal.runners.MethodRoadie.runBefores(MethodRoadie.java:133)
	at org.junit.internal.runners.MethodRoadie.runBeforesThenTestThenAfters(MethodRoadie.java:96)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.executeTest(PowerMockJUnit44RunnerDelegateImpl.java:300)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.executeTestInSuper(PowerMockJUnit47RunnerDelegateImpl.java:131)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.access$100(PowerMockJUnit47RunnerDelegateImpl.java:59)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner$TestExecutorStatement.evaluate(PowerMockJUnit47RunnerDelegateImpl.java:147)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.evaluateStatement(PowerMockJUnit47RunnerDelegateImpl.java:107)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit47RunnerDelegateImpl$PowerMockJUnit47MethodRunner.executeTest(PowerMockJUnit47RunnerDelegateImpl.java:82)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$PowerMockJUnit44MethodRunner.runBeforesThenTestThenAfters(PowerMockJUnit44RunnerDelegateImpl.java:288)
	at org.junit.internal.runners.MethodRoadie.runTest(MethodRoadie.java:87)
	at org.junit.internal.runners.MethodRoadie.run(MethodRoadie.java:50)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.invokeTestMethod(PowerMockJUnit44RunnerDelegateImpl.java:208)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.runMethods(PowerMockJUnit44RunnerDelegateImpl.java:147)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl$1.run(PowerMockJUnit44RunnerDelegateImpl.java:121)
	at org.junit.internal.runners.ClassRoadie.runUnprotected(ClassRoadie.java:34)
	at org.junit.internal.runners.ClassRoadie.runProtected(ClassRoadie.java:44)
	at org.powermock.modules.junit4.internal.impl.PowerMockJUnit44RunnerDelegateImpl.run(PowerMockJUnit44RunnerDelegateImpl.java:123)
	at org.powermock.modules.junit4.common.internal.impl.JUnit4TestSuiteChunkerImpl.run(JUnit4TestSuiteChunkerImpl.java:121)
	at org.powermock.modules.junit4.common.internal.impl.AbstractCommonPowerMockRunner.run(AbstractCommonPowerMockRunner.java:53)
	at org.powermock.modules.junit4.PowerMockRunner.run(PowerMockRunner.java:59)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:538)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:760)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:206)
Caused by: java.lang.ClassNotFoundException: org.joda.time.ReadablePeriod
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.powermock.core.classloader.MockClassLoader.loadModifiedClass(MockClassLoader.java:190)
	at org.powermock.core.classloader.DeferSupportingClassLoader.loadClass(DeferSupportingClassLoader.java:71)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 43 more

File: common-util/src/main/java/org/sunbird/common/models/util/ActorOperations.java (hunk 2 above)
Comment (amolnin) [reply common-util/src/main/java/org/sunbird/common/models/util/ActorOperations.java]: Yes, I am going to re-merge my changes and put another pull-request without such discrepancies.

**Pull request closed without merging**